modules/distribution/pom.xml

```diff
@@ -63,6 +63,9 @@
     <plugins>
       <plugin>
         <artifactId>maven-assembly-plugin</artifactId>
+        <configuration>
+          <escapeString>\</escapeString>
```

Comment:
This config should be moved into the execution-specific config a few lines lower.